proposals/Level-1.md

```diff
@@ -202,27 +180,30 @@ associated with the exception index. These values are popped of the stack and
 are used (along with the corresponding exception tag) to create the
 corresponding exception value. That exception value is then thrown.
 
-When an exception value is thrown, the host VM searches for nearest enclosing
-try block body that execution is in. That try block is called the _catching_ try
-block.
+When an exception value is thrown, the embedder searches for the nearest
```

Comment:
"the embedder searches" - it's the Wasm VM. How about: "When an exception is thrown, the active `try` block entered last is called the _catching_ try block."